hip/test_wrapper_thapilog.sh.in

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+: "${CC:=gcc}"
+
+btx_model_path="@builddir@/btx_hip_model.yaml"
+trace_path="$1"
+trace_name="$(basename $trace_path)"
+trace_name="${trace_name%.thapilog}"
+expected_output_path="$(dirname $trace_path)/${trace_name}.bt2out"
```

Comment:
should we pass it at `$2` to be less magical or more explicit?

Reply:
I am not sure how to pass more args to the autotools test framework "Log compilers". I think there is a special variable for it, but not sure deriving the bath in awkward make syntax and using the magic variable to pass an extra arg is simpler vs deriving the path in the script?

Reply:
Yes true. Maybe too complicated :( . Just make stuff a little too much implicit for my taste right now, but i can live with it!